web/modules/custom/rdf_entity/rdf_entity.module

```diff
@@ -69,22 +69,35 @@ function rdf_entity_form_field_storage_config_edit_form_alter(&$form, FormStateI
       '#title' => $column,
       '#description' => $description,
       '#weight' => 150,
-      '#default_value' => $entity->getThirdPartySetting('rdf_entity', 'mapping_' . $column, FALSE),
+      '#default_value' => rdf_entity_get_third_party_property($entity, 'mapping', $column, FALSE),
     );
   }
 
   $form['#entity_builders'][] = 'rdf_entity_form_alter_builder';
 }
 
 /**
+ * Retrieve nested third party settings from object.
+ */
+function rdf_entity_get_third_party_property(ConfigEntityInterface $object, $property, $column, $default = NULL) {
```

Comment:
Do you think that this could go to the mapping handler as 'get_third_party_property'?